spock/plugins/helpers/inventory.py

```diff
@@ -78,44 +85,58 @@ def __repr__(self):
         if self.item_id == INV_ITEMID_EMPTY:
             args = 'empty'
         else:
-            args = str(self.get_dict()).strip('{}').replace("'", '').replace(': ', '=')
-        return 'Slot(window=%s, slot_nr=%i, %s)' % (self.window, self.slot_nr, args)
+            args = str(self.get_dict()).strip('{}').replace("'", '').replace(
+                ': ', '=')
+        return 'Slot(window=%s, slot_nr=%i, %s)' % (
+            self.window, self.slot_nr, args)
+
 
 class SlotCursor(Slot):
     def __init__(self, id=INV_ITEMID_EMPTY, damage=0, amount=0, enchants=None):
         class CursorWindow:
             window_id = INV_WINID_CURSOR
+
             def __repr__(self):
                 return 'CursorWindow()'
-        super(self.__class__, self).__init__(CursorWindow(), INV_SLOT_NR_CURSOR, id, damage, amount, enchants)
+
+        super(self.__class__, self).__init__(CursorWindow(),
+                                             INV_SLOT_NR_CURSOR, id, damage,
+                                             amount, enchants)
 
 # look up a class by window type ID when opening windows
 inv_types = {}
+
+
 def map_window_type(inv_type_id):
     def inner(cl):
         inv_types[inv_type_id] = cl
         cl.inv_type = inv_type_id
         return cl
+
     return inner
 
+
 class InventoryBase(object):
     """ Base class for all inventory types. """
 
     # the arguments must have the same names as the keys in the packet dict
-    def __init__(self, inv_type, window_id, title, slot_count, persistent_slots):
+    def __init__(self, inv_type, window_id, title, slot_count,
+                 persistent_slots):
         self.inv_type = inv_type
         self.window_id = window_id
         self.title = title
 
-        # slots vary by inventory type, but always contain main inventory and hotbar
-        # create own slots, ...
+        # slots vary by inventory type, but always contain main inventory and
+        # hotbar create own slots, ...
```

Comment:
No idea what this means, I haven't actually started reading code, just preparing it to start reading